noetic/distribution.yaml

```diff
@@ -1242,6 +1242,24 @@ repositories:
       url: https://github.com/OTL/cv_camera.git
       version: master
     status: maintained
+  darknet_ros:
+    doc:
+      type: git
+      url: https://github.com/leggedrobotics/darknet_ros.git
+      version: master
+    release:
+      packages:
+      - darknet_ros_msgs
```

Comment:
@mbjelonic hmm, isn't `darknet_ros` missing? This doesn't match what can be found in https://github.com/leggedrobotics/darknet_ros-release. Did you use `bloom`  for this?

Reply:
Yes, I use bloom but I always have to adapt it because sadly I can not add darknet to your build system since it does not allow for submodules.

Check out our last release in kinetic (it is done in a similar way): https://github.com/ros/rosdistro/blob/master/kinetic/distribution.yaml

Reply:
> your build system since it does not allow for submodules

I'm almost certain that Bloom automatically pulls all submodules in a repository prior to importing, so the submodule directories should be present and filled on the buildfarm. The one caveat is that they're no longer submodules, but are just subdirectories, so you can't interact with them using git.

Reply:
I added the package darknet_ros back. Let's see what the ROS build server gives me now.

Reply:
Ok seems like it passed. I remember that a few years ago, I had issues with adding the darknet_ros package. Thanks for the help!